fabfile.py

```diff
@@ -36,17 +36,17 @@ def build_static():
     local("./node_modules/.bin/gulp")
 
 
-def clear_memcached():
+def clear_redis_cache():
     try:
-        cache.init(config.MEMCACHED_SERVERS)
+        cache.init(config.REDIS_HOST, config.REDIS_PORT, config.REDIS_NAMESPACE)
```

Comment:
This is hard to argue with. I'll add a `ns_versions_loc` parameter as well. What do you think should be the directory path/name for namespace?